[Bug] Release build command in docs fails with @testable import error

## Bug

The documented release build command `tuist build -- -configuration Release` fails because it builds all schemes including test targets, which use `@testable import` that requires `-enable-testing` (only available in Debug configuration).

## Environment
- **ClaudeBar**: v1.0.0
- **macOS**: 15.x
- **Tuist**: latest

## Steps to Reproduce

1. Clone the repo
2. Run `tuist install`
3. Run `tuist build -- -configuration Release`

## Expected Behavior

Release build completes successfully.

## Actual Behavior

Build fails with:

```
❌ /Users/.../Tests/AcceptanceTests/ActionBarSpec.swift:4:18: module 'Domain' was not compiled for testing

warning: ignore swiftmodule built without '-enable-testing':
  .../Build/Products/Release/Domain.framework/Modules/Domain.swiftmodule/arm64-apple-macos.swiftmodule
warning: ignore swiftmodule built without '-enable-testing':
  .../Build/Products/Release/Infrastructure.framework/Modules/Infrastructure.swiftmodule/arm64-apple-macos.swiftmodule

** BUILD FAILED **
  CompileSwift normal arm64 (in target 'AcceptanceTests' from project 'ClaudeBar')
```

Additionally, moving `-scheme` after `--` also fails:

```
✖ Error
  The argument -scheme added after the terminator (--) cannot be passed through to
  xcodebuild because it is handled by Tuist.
```

## Fix

Use tuist's native flags instead of passing through to xcodebuild:

```bash
tuist build ClaudeBar -C Release
```

This targets only the app scheme and uses tuist's built-in `-C` (configuration) flag.

--- a/CLAUDE.md
+++ b/CLAUDE.md
@@ -40,7 +40,7 @@ tuist test DomainTests
 tuist test --result-bundle-path TestResults.xcresult -- -enableCodeCoverage YES
 
 # Build release configuration
-tuist build -- -configuration Release
+tuist build ClaudeBar -C Release
 ```
 
 **Key files:**

--- a/README.md
+++ b/README.md
@@ -99,7 +99,7 @@ brew install tuist
 
 # Install dependencies and build
 tuist install
-tuist build -- -configuration Release
+tuist build ClaudeBar -C Release
 ```
 
 ## Usage
@@ -144,7 +144,7 @@ tuist test
 tuist test --result-bundle-path TestResults.xcresult -- -enableCodeCoverage YES
 
 # Build release configuration
-tuist build -- -configuration Release
+tuist build ClaudeBar -C Release
 ```
 
 ### SwiftUI Previews

